addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/RFXComValueSelector.java

```diff
@@ -64,7 +65,7 @@
     private final String text;
     private Class<? extends Item> itemClass;
 
-    private RFXComValueSelector(final String text, Class<? extends Item> itemClass) {
+    RFXComValueSelector(final String text, Class<? extends Item> itemClass) {
```

Comment:
Why the visibility change here? I don't see any other changes using this constructor from elsewhere.